lib/Models/ActionModel.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * MoesifAPILib
+ *
+ * transactionId: Option[String],
+ * actionName: String,
+ * sessionToken: Option[String],
+ * userId: String,
+ * companyId: Option[String],
+ * metadata: Option[JsObject],
+ * request: Option[JsObject]
+ */
+var BaseModel = require("./BaseModel");
+/**
+ * Creates a instance of ActionModel
+ *
+ * @constructor
+ */
+var ActionModel = function (obj) {
+  if (!obj) {
+    this.transactionId = null;
+    this.actionName = null;
+    this.sessionToken = null;
+    this.userId = null;
+    this.companyId = null;
+    this.metadata = null;
+    this.request = null;
+  } else {
+    this.transactionId = obj.transactionId;
+    this.actionName = obj.actionName;
+    this.sessionToken = obj.sessionToken;
+    this.userId = obj.userId;
+    this.companyId = obj.companyId;
+    this.metadata = obj.metadata;
+    this.request = obj.request;
+  }
+
+  // Append to variable dictionary
+  this._variableDict["userId"] = "user_id";
+  this._variableDict["companyId"] = "company_id";
+  this._variableDict["sessionToken"] = "session_token";
+};
```

Comment:
are you sure `user_agent_string` and `ip_address` doesn't need to be in the dictionary here? for handling translating form camel case to snake case?   Is it carried over in the payload without?

Reply:
I don't remember if it is is the right place to add it, just double check the { request.userAgentString  and request.ipAddress is translated correctly snake_case for the final payload.